src/buildHTML.js

```diff
@@ -547,14 +547,40 @@ groupTypes.genfrac = function(group, options) {
         options);
 };
 
+/**
+ * Parse a `sizeValue`, as parsed by functions.js argType "size", into
+ * a CSS em value. `options` gives the current options.
+ */
 const calculateSize = function(sizeValue, options) {
-    let x = sizeValue.number;
-    if (sizeValue.unit === "ex") {
-        x *= options.fontMetrics().emPerEx;
-    } else if (sizeValue.unit === "mu") {
-        x /= 18;
+    let scale;
+    // `mu` units scale with scriptstyle/scriptscriptstyle.
+    // Other units always refer to the *textstyle* font in the current size.
```

Comment:
Below it looks like we're only using textstyle if the current style `isTight`.  If that's the correct behavior could you update this comment to reflect that?